packages/bolt-connection/src/connection/connection-channel.js

```diff
@@ -393,7 +394,27 @@ export default class ChannelConnection extends Connection {
   }
 
   /**
-   * This method still here because it's used by the {@link PooledConnectionProvider}
+   * This method is used by the {@link PooledConnectionProvider}
+   *
+   * @param {any} observer
+   */
+  _setIdle (observer) {
+    this._idle = true
+    this._ch.stopReceiveTimeout()
+    return this._protocol.queueObserverIfProtocolIsNotBroken(observer)
```

Comment:
This return value isn't being used, is it?

Reply:
It isn't.